sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/ApplicationDetailsForCOE.vue

```diff
@@ -253,6 +259,7 @@ export default defineComponent({
     });
 
     return {
+      isReadOnlyUser,
```

Comment:
Thanks for making this change. I belive the `isReadOnlyUser` is no longer needed in the return and can be removed.